packages/foreman/foreman-selinux/foreman-selinux.spec

```diff
@@ -60,7 +58,7 @@ Requires(postun):   /usr/sbin/semodule, /sbin/restorecon
 SELinux policy module for Foreman
 
 %prep
-%setup -q -n %{name}-%{version}%{?dashalphatag}
+%setup -q -n %{name}-%{version}%{?prerelease:.}%{?prerelease}
```

Comment:
The build is failing because this needs to be a dash. Can't the setup macro figure this out itself from source0?